src/pages/usage/getting-started.md

```diff
@@ -30,8 +30,12 @@ Port-forward to an ACD pod. **Note that only an admin can do the port-forward an
 oc port-forward ${pod_name} -n ${acd_namespace} 9443:9443
 ```
 
+The terminal should hang here waiting to accept network traffic on the localhost port 9443.  Open a new terminal window to run the following steps.
+
 #### 3. Use curl to deploy the cartridge.
 
+In a new terminal window run one the following commands to deploy the cartridge.  Be sure to be in the directory where the cartridge zip file was downloaded too.
```

Comment:
Drop "too" at the end, or add "to which" and drop "where" in phrase "where the cartridge zip file was downloaded."